packages/compass-e2e-tests/index.js

```diff
@@ -19,6 +19,13 @@ function setup() {
   debug('Starting MongoDB server and importing fixtures');
   spawnSync('npm', ['run', 'start-server'], { stdio: 'inherit' });
   spawnSync('npm', ['run', 'insert-data'], { stdio: 'inherit' });
+
+  try {
+    debug('Clearing out past logs');
+    fs.rmdirSync('.log', { recursive: true });
```

Comment:
This is making me all sorts of tingly … can we compute `.log` relative to `__dirname`, as it’s done in the other file? :smile: